assembly/assembly-wsmaster-war/src/main/webapp/WEB-INF/classes/che/che.properties

```diff
@@ -12,117 +12,117 @@
 che.workspace.projects.storage=/projects
 
-# Used when devfile k8s/os type components requests project PVC creation
-# (applied in case of unique and perWorkspace PVC strategy. In case of common PVC strategy,
-# it will be rewritten with value of che.infra.kubernetes.pvc.quantity property)
+# Used when Kubernetes or OpenShift-type components in a devfile request project PVC creation
+# (Applied in case of 'unique' and 'per workspace' PVC strategy. In case of the 'common' PVC strategy,
+# it is rewritten with the value of the `che.infra.kubernetes.pvc.quantity` property.)
 che.workspace.projects.storage.default.size=1Gi
 
 # Defines the directory inside the machine where all the workspace logs are placed.
-# The value of this folder should be provided into machine e.g. like environment variable
-# so agents developers can use this directory for backup agents logs.
+# Provide this value into the machine, for example, as an environment variable.
+# This is to ensure that agent developers can use this directory to back up agent logs.
 che.workspace.logs.root_dir=/workspace_logs
 
-# Configures proxies used by runtimes powering workspaces
+# Configures proxies used by runtimes powering workspaces.
 che.workspace.http_proxy=
 che.workspace.https_proxy=
 che.workspace.no_proxy=
 
-# By default, when users access to a workspace with its URL the workspace
-# automatically starts if it is stopped. You can set this to false to disable this.
+# By default, when users access a workspace with its URL, the workspace
+# automatically starts (if currently stopped). Set this to `false` to disable this behaviour.
 che.workspace.auto_start=true
 
-# Workspace threads pool configuration, this pool is used for workspace related
-# operations that require asynchronous execution e.g. starting/stopping.
-# Possible values are 'fixed', 'cached'
+# Workspace threads pool configuration. This pool is used for workspace-related
+# operations that require asynchronous execution, for example, starting and stopping.
+# Possible values are `fixed` and `cached`.
 che.workspace.pool.type=fixed
 
-# This property is ignored when pool type is different from 'fixed'.
-# Configures the exact size of the pool, if it's set multiplier property is ignored.
-# If this property is not set(0, < 0, NULL) then pool sized to number of cores,
-# it can be modified within multiplier
+# This property is ignored when pool type is different from `fixed`.
+# It configures the exact size of the pool. When set, the `multiplier` property is ignored.
+# If this property is not set (`0`, `<0`, `NULL`), then the pool size equals the number of cores.
+# It can be modified by the `multiplier`.
 che.workspace.pool.exact_size=30
 
-# This property is ignored when pool type is different from 'fixed' or exact pool size is set.
-# If it's set the pool size will be N_CORES * multiplier
+# This property is ignored when pool type is different from `fixed` or an exact pool size is set.
```

Comment:
Should this be phrased "This property is ignored when pool type not set to `fixed`, or `che.workspace.pool.exact_size` is set."